dataproc/hail_demo_dashboard.md

```diff
@@ -8,7 +8,17 @@ The example specifically annotates GWAS results from the paper [*Whole genome se
 
 # How do I get started?
 
-*Duplicate* this workspace. You can do that via the 'three-dot' menu in the upper right of the workspace.
+## Step 0: Set up your github SSH key and _Duplicate_ this workspace
+
+- In the Enterprise Terra UI, click on your profile icon, then click on "My Profile". Copy the public SSH key listed there.
+  Then, [add that key](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account) to your GitHub account.
+  This page: https://terra-docs.api.verily.com/docs/how_to_guides/terra_ssh_key_guide/ has more detail.
```

Comment:
I'd make this a markdown link.  e.g. 

See [this page](https://terra-docs.api.verily.com/docs/how_to_guides/terra_ssh_key_guide/) for a more details.

Reply:
done